src/layouts/login-form.html

```diff
@@ -50,7 +50,7 @@
       <img src="/static/icons/favicon-192x192.png" height="192" />
       <a href="#" id="hideKeyboardOnFocus"></a>
       <div class='interact'>
-        <div id='loginform' hidden$="[[showLoading]]">
+        <form id='loginform' hidden$="[[showLoading]]">
```

Comment:
This is out of scope for this PR, however this would probably fix the saving password form dialog on Firefox, so I'm fine with leaving it in.